buidl/taproot.py

```diff
@@ -265,15 +271,40 @@ def __init__(self, points, locktime=None, sequence=None):
 
     def get_tweak_point(self, tweak):
         if self.point.parity:
-            return -1 * self.point + tweak
+            tweak_point = -1 * self.point + tweak
         else:
-            return self.point + tweak
+            tweak_point = self.point + tweak
+        return tweak_point
```

Comment:
Nit: no need to define `tweak_point` if all you're doing is returning it (just return it above).

Reply:
right, I forgot to change this back when I was debugging.